def_files/CHARMM36UA_DPPC.def

```diff
@@ -0,0 +1,58 @@
+palmitoyl_sn2_3a DPPC C23 H3R
```

Comment:
I thought the name was "palmitoyl1" for sn1 et "palmitoyl2" for sn2?
That's what i did for the others

Reply:
Right! I wrote sn1 and sn2 because I found it clear 😉 but it's important to be consistent 👍 We can see later how to update the generic names to have them as clear as possible.